### PR TITLE
Feature/register fix

### PR DESCRIPTION
**Pull request by HayluMulu**
- :hammer: fix Register
- :art: added constants folder that contain consts,enum etc.
- 🚚 renamed Register => SignUp
- 🚚 renamed LoginScreen => Login
- 🚚 renamed AppendDive => AddDive

### CODE REVIEW

**Comment by HayluMulu** [comment]
Done by The Notorious M.U.L.U

File: src/screens/AddDive.tsx [hunk 1]
```diff
@@ -1,27 +1,26 @@
...
     } else {
-      console.log('No user is signed in');
+      console.log('No user is signed in'); // needs logs here 
     }
   }, []);
 
   return (
     <View style={styles.container}>
       <View style={styles.topPortion}>
         {/* <HomeNavbar /> */}
```
[the remark this answers is not in the document]
Comment (drChaos6969) [reply src/screens/AddDive.tsx:22]: and remove that useless component

File: src/screens/AddDive.tsx [hunk 2]
```diff
@@ -1,27 +1,26 @@
...
-import { BottomAddDive } from '../components';
+import { BottomAddDive, HomeNavbar } from '../components';
 import React, { useEffect, useState } from 'react';
 import { View, Text, StyleSheet } from 'react-native';
+import { SafeAreaView } from 'react-native-safe-area-context';
 
-export const AppendDive = () => {
-  const [email, setEmail] = useState<string>('');
+export const AddDive = () => {
+  const [displayName, setDisaplayName] = useState<string>("Guest");
```
[the remark this answers is not in the document]
Comment (HayluMulu) [reply src/screens/AddDive.tsx:8]: no there is no username it its just the firstname of the "daver"

**Comment by HayluMulu** [comment]
Done by The Notorious M.U.L.U

**Pull request merged**